src/vm-memory/src/lib.rs

```diff
@@ -155,3 +155,205 @@ pub mod test_utils {
         GuestMemoryMmap::from_regions(mmap_regions)
     }
 }
+
+#[cfg(test)]
```

Comment:
Can we move this commit right after the first one ? Or in the first one. Since it seems to add tests for the methods that are defined there.